### ISSUE TITLE
JSON output malformed (edge case)

### ISSUE THREAD

**Issue by apeltzer** [apeltzer/DeDup#13]
```
{
  "metadata": {
    "tool_name": "DeDup",
    "version": "0.12.7",
    "sample_name": "SKO719A174.bam"
  },
  "metrics": {
    "total_reads": 26,
    "mapped_reads": 0,
    "merged_removed": 0,
    "forward_removed": 0,
    "total_removed": 0,
    "reverse_removed": 0,
    "dup_rate": "�",
    "clusterfactor": "�"
  }
}
```

reported by Alexandre Gilardet in Nf-core slack: 

https://nfcore.slack.com/archives/CEQBS091V/p1608462786005800?thread_ts=1608302934.003400&cid=CEQBS091V

**Comment by apeltzer** [comment]
@jfy133 - this is happening when mapped == 0, probably some weird numeric error.

**Comment by apeltzer** [comment]
Fixed in 07d47868f10a6830da8c9161caa3755d9da155bf
